src/Shared/Recruit.Vacancies.Client/Infrastructure/Mongo/MongoDbCollectionBase.cs

```diff
@@ -43,6 +42,19 @@ protected IMongoCollection<T> GetCollection<T>()
             return collection;
         }
 
+        protected ProjectionDefinition<T> GetProjection<T>()
+        {
+            ProjectionDefinition<T> projection = null;
```

Comment:
Would this cope with documents with nested complex types in properties? Also will we always need the "Id" field.

Reply:
Nope. This just does simple top level properties which is all we need for now.  Anything more complex you'd probably want to write your own projection for anyway.  As long as the type you use contains an Id field then it will be populated